Selected wallpaper check: follow accent color

lenemter opened the pull request:
Closes #274 

Uses radio button instead of icon

![image](https://user-images.githubusercontent.com/80143868/183424256-a245c1bc-8654-4d51-88cb-886e10d7e3e9.png)

jeremypw reviewed (changes requested):
@lenemter Is it possible to find a solution using Gtk.CheckButton?  Gtk.RadioButton is deprecated and not present in Gtk4.

jeremypw commented:
@lenemter On further thought, I am not sure whether a button is appropriate as the tick is really an emblem - it is not an active button that can be toggled.  Maybe better to color the image?

lenemter commented:
So, I tried to color an image. I took selection-checked icon, removed background from svg and added background in css. Result:

![image](https://user-images.githubusercontent.com/80143868/183561685-4f449208-fd6d-457d-9795-6714f6acabc5.png)

 What do you think looks better?
